sys/net/gnrc/application_layer/dhcpv6/client.c

```diff
@@ -89,41 +89,17 @@ void dhcpv6_client_conf_prefix(unsigned iface, const ipv6_addr_t *pfx,
     }
 }
 
-bool dhcpv6_client_check_ia_na(unsigned iface)
+void dhcpv6_client_conf_addr(unsigned netif, const ipv6_addr_t *addr,
+                             uint32_t valid, uint32_t pref)
 {
-    gnrc_netif_t *netif = gnrc_netif_get_by_pid(iface);
-
-    return netif->ipv6.aac_mode & GNRC_NETIF_AAC_DHCP;
+    dhcpv6_client_conf_prefix(netif, addr, IPV6_ADDR_BIT_LEN, valid, pref);
```

Comment:
I think it's safe to assume that this function will always be implemented like this by the stacks, so I see it more as a convenience function. So...